python/data/ArtusConfigs/Run2LegacyAnalysis_base.py

```diff
@@ -32,7 +31,8 @@ def build_config(nickname, **kwargs):
   isDY = re.search("DY.?JetsToLLM(10to50|50)", nickname)
   isWjets = re.search("W.?JetsToLNu", nickname)
   isSUSYggH = re.search("SUSYGluGluToHToTauTau", nickname)
-  isSignal = re.search("HToTauTau",nickname)
+  isSignal = re.search("HToTauTau|125tautau",nickname)
```

Comment:
Ist es Absicht, dass isSignal hier anders erweitert wird als in den channel configs?

Reply:
fixed